### PR TITLE
Montag vormittag

### PR DESCRIPTION
**Pull request by DominicWebDev**
kleine Text was Montag vormittag gemacht wurde.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,6 @@ Test
 Test
 
 TEST
+
+Am Montag vormittag haben wir ein Kennenlernen gehabt
```
Comment (AnjaSiewert) [inline README.md:7]: I would add: "Anja was my favorite member of the group" 🥰

**Review by AnjaSiewert — APPROVED** [review]
LGTM 🎓 test

File: README.md (hunk 1 above)
Comment (DominicWebDev) [reply README.md:7]: 👯

File: README.md [hunk 2]
```diff
@@ -3,3 +3,6 @@ Test
 Test
```
Comment (Satttoshi) [inline README.md:3]: I would remove the Test messages

File: README.md [hunk 3]
```diff
@@ -3,3 +3,6 @@ Test
...
 
 TEST
```
Comment (Satttoshi) [inline README.md:5]: this one too

**Review by ChristophObst — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Satttoshi) [reply README.md:7]: Test 123

**Review by Satttoshi — APPROVED** [review]
very nice yay

**Pull request merged**